### ISSUE TITLE
PlexPy not launching Web page anymore...just installed.

### ISSUE THREAD

**Issue by KYablonsky** [Tautulli/Tautulli#1215]
I installed PlexPy from the ReadyNAS page here:
https://apps.readynas.com/pages/

From External.

The install worked just fine...took a few minutes, but did install and started with no issues. It even launched the Web page. I then proceeded to look around at the settings, then was ready to proceed wot the FAQ to read up. I noticed that the URL was blank and added the IP address for the Plex server and that appeared to add successfully. I then noticed another URL that was blank or rather had 0.0.0.0...can't remember at this point and once I saved, I am no longer able to access the webpage.

Got home and now I'm internal to the server and stopped the PlexPy service and restarted...same result with internal launch. I then stopped and removed the PlexPy and then re-installed from the same URL above. It installed just fine and when I try to launch it, I get the same results.

I have ReadyNAS 516
Very new to Plex and this whole thing so I really don't have any ideas as to versions of just about anything.

I spent the better part of this last weekend getting Plex and such installed...Now I'm stuck on this.

Any suggestions?
If you want me to provide logs and such, I'll need guidance on commands to get them.

Is there any way to remove FULLY, so that I can install fresh and start from the beginning?

**Comment by JonnyWong16** [comment]
You shouldn't have touched the `HTTP Host` (the 0.0.0.0).

https://github.com/JonnyWong16/plexpy/wiki/Frequently-Asked-Questions-(FAQ)#general-q4

Or delete the config.ini file to start over. I have no idea where the config.ini would be stored on ReadyNAS.

**Comment by KYablonsky** [comment]
I found that I could not find the Config.ini file anywhere on my server. There must be a hidden folder that I could not find where the install goes to. So the install from https://apps.readynas.com/pages/ puts things in an abnormal place I guess.

So, I finally gave up trying to find the config file and removed the PlexPy app and then re-installed using the Ubuntu install instructions from this site. The install went off without a hitch and I was able to launch the web page. When I started through the initial setup, there was a small hiccup where it could not locate my Plex server, but I searched and found a suggestion to restart the service, which I did and then began the initial setup again, and this time the Plex server was found and install was full successful.

This issue is resolved.
